libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -390,6 +390,12 @@ bool AP_Landing::type_slope_is_flaring(void) const
     return (type_slope_stage == SlopeStage::FINAL);
 }
 
+bool AP_Landing::type_slope_is_on_final(void) const
+{
+    return (type_slope_stage == SlopeStage::PREFLARE ||
```

Comment:
in practice this will only impact PREFLARE, as after flare (ie. in FINAL) throttle control is disabled
maybe we should allow lower speed in the approach as long as lined up?